content/en/docs/instrumentation/go/libraries.md

```diff
@@ -18,87 +17,20 @@ track inbound and outbound requests once you configure it in your code.
 ## Setup
 
 Each instrumentation library is a package. In general, this means you need to
-`go get` the appropriate package:
+`go get` the appropriate package. For example, for the instrumentation libraries
+maintained in the
+[Contrib repository](https://github.com/open-telemetry/opentelemetry-go-contrib)
+this means:
```

Comment:
nit: try to avoid repeated use of "for"

```suggestion
`go get` the appropriate package. For example, to get the instrumentation libraries
maintained in the
[Contrib repository](https://github.com/open-telemetry/opentelemetry-go-contrib)
run the following:
```